src/middlewares/errorMiddleware.js

```diff
@@ -0,0 +1,22 @@
+const ApiError = require('../exeptions/apiError.js');
+
+const errorMiddleware = (error, req, res, next) => {
+  if (error instanceof ApiError) {
+    res.status(error.status).send({
+      message: error.message,
+      errors: error.errors,
+    });
+
+    return;
+  }
+
+  res.statusCode = 500;
```

Comment:
Consider using `res.status(500)` instead of directly setting `res.statusCode` for consistency with the rest of the code where `res.status()` is used.